sql/_23_apricot_qa/_01_sql_extension3/_06_merge_statement/cases/_20_adhoc_merge_1.sql

```diff
@@ -138,7 +138,7 @@ values(t1.a, t1.b,t1.c);
 
 merge into t2
 using t1
-on ((t1.a=t2.a) || (t1.a!=t2.a))
+on ((t1.a=t2.a)<>0 || (t1.a!=t2.a))<>0
```

Comment:
The corrections you made are not a problem, but it seems more appropriate to modify them as follows.
`on ((t1.a=t2.a) || (t1.a!=t2.a))<>0`